docs/resources/kubernetes_secret.md

```diff
@@ -0,0 +1,121 @@
+---
+Title: "Kubernetes Secret Resource"
+Description: |-
+    Creating the Kubernetes Secret resource.
+---
+
+# Kubernetes Secret
+
+Manage a Kubernetes Secret using this Terraform module.
+
+In managed clusters, both attached and provisioned, you can create Kubernetes Secret that you can manage through Tanzu Mission Control.
+
+To create a cluster secret, you must be associated with the cluster.admin role
+
+[cluster Secret]: https://docs.vmware.com/en/VMware-Tanzu-Mission-Control/services/tanzumc-using/GUID-BBE2404D-C2EE-41C7-B639-C0322783A74D.html
```

Comment:
Please add another reference for secret export from the TMC documentation.

Reply:
added PTAL